cas/src/main/java/org/springframework/security/cas/authentication/CasAuthenticationToken.java

```diff
@@ -57,15 +59,18 @@
      *        org.springframework.security.core.userdetails.UserDetailsService}) (cannot be <code>null</code>)
      * @param assertion the assertion returned from the CAS servers.  It contains the principal and how to obtain a
      *        proxy ticket for the user.
+     * @param rememberMeAttributeName the CAS remember-me attribute name
      *
      * @throws IllegalArgumentException if a <code>null</code> was passed
      */
     public CasAuthenticationToken(final String key, final Object principal, final Object credentials,
-        final Collection<? extends GrantedAuthority> authorities, final UserDetails userDetails, final Assertion assertion) {
+        final Collection<? extends GrantedAuthority> authorities, final UserDetails userDetails, final Assertion assertion,
+        String rememberMeAttributeName) {
         super(authorities);
 
         if ((key == null) || ("".equals(key)) || (principal == null) || "".equals(principal) || (credentials == null)
```

Comment:
Instead of this long if for parameter validations could be used 
Assert.notNull(myParameter, "myParamter must not be null"); which is a replacement for already java 8 Objects.requireNonNull()